moveit_ros/robot_interaction/src/kinematic_options.cpp

```diff
@@ -55,7 +55,9 @@ bool robot_interaction::KinematicOptions::setStateFromIK(robot_state::RobotState
     ROS_ERROR("No getJointModelGroup('%s') found", group.c_str());
     return false;
   }
-  bool result = state.setFromIK(jmg, pose, tip, max_attempts_, timeout_seconds_, state_validity_callback_, options_);
+  bool result = state.setFromIK(jmg, pose, tip, max_attempts_,
+                                timeout_seconds_ > 0.0 ? timeout_seconds_ : std::min(0.1, jmg->getDefaultIKTimeout()),
```

Comment:
It strikes me as odd that there are two minimum values in this statement - even though one is intended to be default. If I am reading it correctly, the `JointModelGroup`'s default IK timeout can't be below `0.1` sec, but a time between `(0.0, 0.1]` is possible if the `timeout_seconds_` variable is set to be inside that range. 

Is that intentional, or should the conditional be checking `timeout_seconds_ > 0.1`, with the constructor defaulting to `0.1`?

```
: timeout_seconds_(0.1) // 0.1 = use default timeout
```